.github/workflows/deploy-to-control-plane.yml

```
@@ -140,33 +144,62 @@
+              deploymentId: deployment.data.id,
+              commentId: comment.data.id,
+              workflowUrl
+            };
+
+      - name: Set comment ID and workflow URL
+        run: |
+          echo "COMMENT_ID=${{ fromJSON(steps.init-deployment.outputs.result).commentId }}" >> $GITHUB_ENV
+          echo "WORKFLOW_URL=${{ fromJSON(steps.init-deployment.outputs.result).workflowUrl }}" >> $GITHUB_ENV
+
+      - name: Set commit hash
+        run: |
```

actionlint (1.7.4)

property "getref" is not defined in object type {check-app: {conclusion: string; outcome: string; outputs: {string => string}}; create-comment: {conclusion: string; outcome: string; outputs: object}; init-deployment: {conclusion: string; outcome: string; outputs: object}; workflow-url: {conclusion: string; outcome: string; outputs: object}}

(expression)